Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -105,6 +105,7 @@ extension Command {
         var args = module.basicArgs
         args += module.optimizationFlags(conf)
         args += ["-L\(prefix)"]
+        args += module.languageArgs
```

Comment:
ah cool, I modified language args to search for any cpp source in dependencies too.